### ISSUE TITLE
Alex, did you stop development on this one? It was very cool.

### ISSUE THREAD

**Issue by Laurentiu-Andronache** [alexbakker/gonano#3]

**Comment by alexbakker** [comment]
Yes, the lack of protocol documentation brought development to a halt and I've started focusing on other things. If anyone would like to continue working on it, feel free to fork it.

Thanks for the kind words, though.
